File: python/iniciante/problem1019.py
# Inputs

t = int(input()) # Seconds

# Calculations

hou = int(t/(60*60))    # Hours
rest = int(t % (60*60))
min = int(rest/60)      # Minutes
sec = int(rest % 60)    # Seconds

# Outputs

print(str(hou) + ":" + str(min) + ":" + str(sec))

File: go/01-iniciante/problem1019.go
package main

import "fmt"

func main() {
    var t int

    fmt.Scanf("%d", &t)

    hou := t/(60*60)
    rest := t % (60*60)
    min := rest/60
    sec := rest % 60

    output := fmt.Sprintf("%d:%d:%d", hou, min, sec)
    fmt.Println(output)
}
